### ISSUE TITLE
Talkback does not present accessibilityLabel on TextField with a value or hint filled in

### ISSUE THREAD

**Issue by oslc15** [Notalib/nativescript-accessibility-ext#51]
### Which platform(s) does your issue occur on?
- Android
- Android 10
- device

### Please, provide the following version numbers that your issue occurs with:

- CLI: 6.5.0
- Cross-platform modules: 6.5.1
- Runtime(s): 6.5.0
- Plugin(s): "@nota/nativescript-accessibility-ext": "6.4.2"

### Please, tell us how to recreate the issue in as much detail as possible. 
Describe the steps to reproduce it.

targeting a filled textfield makes talkback present only value and the information about the type (textfield) like: 
"VALUE" "EDITABLE".
I would like the textfield to be presented with accessibilityLabel and accessibilityHint like:
 "ACCESSIBILITYLABEL" "VALUE" "EDITABLE"

Im not sure that this is by design or a bug. I think it only happens in Android 10.

### Is there any code involved?
This example has three textfields.
The first is without a value and a hint. This works as expected.
The second has a value but no hint. This one does not read the accessibilityLabel on focus.
The third has a hint but no value. This one does not read the accessibilityLabel on focus.

 https://play.nativescript.org/?template=play-vue&id=7MYPwN

**Comment by oslc15** [comment]
@m-abs Another weird behavior is 
`<Label text="demo text" accessible="true" accessibilityLabel="demo text" />`
will read up "demo text demo text" two times in a row. For android 9 and earlier it will only be read once because accessibilityLabel overrides the text attribute

**Comment by m-abs** [comment]
I've done a bit of research.

On Android `hint` and `text` overrides `ContentDescription` for textfields.

We concat `accessibilityLabel`, `accessibilityValue`/`text`/`title` (unless `text` or `title` is equal to `accessibilityLabel`) and `accessibilityHint` into a string that is added to `ContentDescription`.
This value seems to be overridden by android sdk <= 28.

This isn't easy to fix.

In our app we have a `<Label>` just before the `<TextField>` describing it, you could use that to work around this problem.

> <Label text="demo text" accessible="true" accessibilityLabel="demo text" />

The only way I can reproduce that, is by adding a space to one of the values. I can make a quick fix for this, but we don't do anything big before this plugins in migrated to `@nativescript/core`.

**Comment by oslc15** [comment]
@m-abs Ok i see. Our app is in BETA and will be until 2021 so i will add this as a known issue until later. Great news about the `@nativescript/core` migration! is there a date set for the migration yet?

> In our app we have a <Label> just before the <TextField> describing it, you could use that to work around this problem.

I have noticed that android recommends a label before the textfield but it needs the ` android:labelFor` for improved accessibility. Do you use labelFor in some way with the label before the textfield? Our implementation has a label before the textfield but it currently is set to `accessibilityHidden="true"` because i couldnt find a way to use the labelFor yet

**Comment by m-abs** [comment]
> is there a date set for the migration yet?

Sorry, I don't want to make any promises.

I think `android:labelFor` should become an upstream feature in nativescript, but I don't know if I'll get around to implementing it.

**Comment by m-abs** [comment]
It has been merged to `@nativescript/core` and should be available from v8
